starfish/core/spots/DetectPixels/combine_adjacent_features.py

```diff
@@ -212,7 +212,7 @@ def _calculate_mean_pixel_traces(
 
         # the 0th pixel trace corresponds to background. If present, drop it.
         try:
-            mean_pixel_traces = mean_pixel_traces.drop(0, dim=Features.AXIS)
+            mean_pixel_traces = mean_pixel_traces.drop_sel({Features.AXIS: 0})
```

Comment:
@ambrosejcarr it looks like `drop` is being [deprecated](https://github.com/pydata/xarray/pull/3475).  I changed this to `drop_sel`, but there is [no test coverage](https://github.com/spacetx/starfish/blob/0324fbd049de4bbfc9d2b774251d08ebb8a4f2c0/starfish/core/spots/DetectPixels/test/test_calculate_mean_pixel_traces.py#L15) for this code.  can you visually check this to see if it seems sane?